src/scss/main.scss

```diff
@@ -107,6 +107,42 @@ main.container {
   width: auto; margin: 0 2%;
   max-width: 60em;
 
+  .container {
+    text-align: center;
+    width: 80%;
+  }
+  a {
+    text-decoration: none;
+    color: #0192b5;
+  }
+
+  a:hover {
+    text-decoration: underline;
+  }
```

Comment:
Again, a little too general, and better to nest the hover states inside the selector definition, e.g.

```scss
a {
  text-decoration: none;
  color: #0192b5;

  &:hover {
    text-decoration: underline;
  }
}
```